.ci/acceptance-tests/inspec-integration.sh

```diff
@@ -75,7 +64,7 @@ for i in {1..30}
 do
 	# Cleanup cassettes folder each time, we don't want to use a recorded cassette if it records an unauthorized response
 	rm -r inspec-cassettes
-	inspec exec inspec-mm --attrs=attributes/attributes.yaml -t gcp2://
+	inspec exec verify-mm --attrs=attributes/attributes.yaml -t gcp:// --no-distinct-exit
 	if [ "$?" -eq "0" ]; then
```

Comment:
The best way to do this is just

`if inspec exec verify-mm .... ; then`

The current way has an implicit dependency between the two lines which later editing might break.